tests/test_main.py

```diff
@@ -57,7 +57,18 @@ def test_main_multiple_wheels(fancy_wheel, another_fancy_wheel, tmp_path):
 def test_main_prefix(fancy_wheel, tmp_path):
     destdir = tmp_path / "dest"
 
-    main([str(fancy_wheel), "-d", str(destdir), "-p", "/foo"], "python -m installer")
+    main(
```

Comment:
I don't like hijacking the prefix test for that option. Please move your checks to a separate test.